### PR TITLE
Fix typo and spelling inconsistency in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kokifpen**
Fix some typo and spelling inconsistency in CONTRIBUTING.md.

- Unifying `behaviour` and `behavior`
- Fix `modularisation ` to `modularization`
- Fix `GitHub`

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by btcdrak** [comment]
NACK. I am the original author and I deliberately used English spelling.

**Comment by kokifpen** [comment]
@btcdrak Thanks for your comment. In such case, we don't need to change "modularisation" and "behaviour", but we have to change another "behavior", don't we?

**Comment by MarcoFalke** [comment]
We allow both variants of English to be used. Even in the same file.

**Comment by laudaa** [comment]
That's somewhat inconsistent, but I perfectly understand the reasoning behind it. The only valid change is 'GitHub'? If the author is planning on doing more of  these, then they should probably combine them.

**Comment by kokifpen** [comment]
Thanks for discussion about this.
I removed unnecessary changes. And I fixed some GitHub spelling issues in other files.

**Comment by btcdrak** [comment]
Looks good d788f38 :+1:

**Comment by instagibbs** [comment]
ACK, could you squash?

File: doc/translation_process.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Translations
 The Bitcoin-Core project has been designed to support multiple localisations. This makes adding new phrases, and completely new languages easily achievable. For managing all application translations, Bitcoin-Core makes use of the Transifex online translation management tool.
 
 ### Helping to translate (using Transifex)
-Transifex is setup to monitor the Github repo for updates, and when code containing new translations is found, Transifex will process any changes. It may take several hours after a pull-request has been merged, to appear in the Transifex web interface.
+Transifex is setup to monitor the GitHub repo for updates, and when code containing new translations is found, Transifex will process any changes. It may take several hours after a pull-request has been merged, to appear in the Transifex web interface.
```
Comment (paveljanik) [inline doc/translation_process.md]: There is a second instance of Github in this file, please fix it too (https://github.com/bitcoin/bitcoin/pull/9675/files#diff-eb26390875720fc65e1c6c74a87626ffR35).

Comment (kokifpen) [inline doc/translation_process.md]: @paveljanik Thanks for letting me know! I fixed it too.

**Comment by kokifpen** [comment]
Fix one more GitHub spelling.

**Comment by kokifpen** [comment]
@MarcoFalke Done. Thanks for notifying me about squashing.

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/9675/commits/36164faa338d387177c6328496f63ac11eda3b78

**Review by ghost — APPROVED** [review]

**Comment by syaifudin857** [comment]
Join

**Pull request merged**